### PR TITLE
Miscellaneous style guide edits

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

#### Description of Change:

> Edits include the following to align with our style guide:
> - Replaced "i.e.," with "such as"
> - Replaced "e.g.," with "for example,"
> - Removed "ad hoc" and "ad-hoc"
> - Fixed minor typos

 
Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

**Comment by lydia-xie** [comment]
Need to clarify with Core Messaging and Ingestion for how they use the word "ad hoc"

File: _docs/_api/endpoints/user_data/post_user_track.md [hunk 1]
```diff
@@ -395,6 +395,6 @@ When the `X-Braze-Bulk` header is present with any value, Braze will consider th
 Consider the following use cases where you may use the bulk update header:
 
 - A daily job where multiple users' custom attributes are updated via the `/users/track` endpoint.
-- An ad-hoc user data backfill script which updates user information via the `/users/track` endpoint.
+- An impromptu user data backfill script which updates user information via the `/users/track` endpoint.
```
Comment (lydia-xie) [inline _docs/_api/endpoints/user_data/post_user_track.md:398]:
```suggestion
- A user data backfill script that updates user information via the `/users/track` endpoint.
```
Confirmed with Dave from Ingestion

**Review by josh-mccrowell-braze — APPROVED** [review]
Spot checked PR to make sure all content looked good. 
Reviewed types of common mistakes and new style guide changes.
Double checked to make sure any "code owner" level edits were made, and we are clear.

**Comment by lydia-xie** [comment]
> Need to clarify with Core Messaging and Ingestion for how they use the word "ad hoc"

Just waiting on Core Messaging for the "ad hoc" phrasing!

File: _docs/_api/api_limits.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ A single request to the [Messaging endpoints][1] can reach any one of the follow
 
 - Up to 50 specific `external_ids`, each with individual message parameters
 - A segment of any size created in the Braze dashboard, specified by its `segment_id`
-- An ad-hoc audience segment of any size, defined in the request as a [Connected Audience][2] object
+- An impromptu audience segment of any size, defined in the request as a [connected audience][2] object
```
Comment (lydia-xie) [inline _docs/_api/api_limits.md:78]:
```suggestion
- An audience segment of any size, defined in the request as a [connected audience][2] object
```

File: _docs/_api/basics.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ API key permissions are permissions you can assign a user or group to limit thei
 
 | Permission | Endpoint | Description |
 |---|---|---|
-| `messages.send` | [`/messages/send `]({{site.baseurl}}/api/endpoints/messaging/send_messages/post_send_messages/) | Send an immediate, ad-hoc message to specific users. |
+| `messages.send` | [`/messages/send `]({{site.baseurl}}/api/endpoints/messaging/send_messages/post_send_messages/) | Send an immediate, impromptu message to specific users. |
```
Comment (lydia-xie) [inline _docs/_api/basics.md:107]:
```suggestion
| `messages.send` | [`/messages/send `]({{site.baseurl}}/api/endpoints/messaging/send_messages/post_send_messages/) | Send an immediate, one-time message to specific users. |
```

File: _docs/_api/endpoints/messaging/send_messages/post_send_transactional_message.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ description: "This article outlines details about the Send transactional email m
 /transactional/v1/campaigns/{campaign_id}/send
 {% endapimethod %}
 
-> Use this endpoint to send immediate, ad-hoc transactional messages to a designated user. 
+> Use this endpoint to send immediate, impromptu transactional messages to a designated user.
```
Comment (lydia-xie) [inline _docs/_api/endpoints/messaging/send_messages/post_send_transactional_message.md:18]:
```suggestion
> Use this endpoint to send immediate, one-off transactional messages to a designated user. 
```

File: _docs/_api/endpoints/messaging/send_messages/post_send_triggered_campaigns.md [hunk 5]
```diff
@@ -14,7 +14,7 @@ description: "This article outlines details about the Send campaigns via API-tri
 /campaigns/trigger/send
 {% endapimethod %}
 
-> Use this endpoint to send immediate, ad-hoc messages to designated users via API-triggered delivery. 
+> Use this endpoint to send immediate, impromptu messages to designated users via API-triggered delivery.
```
Comment (lydia-xie) [inline _docs/_api/endpoints/messaging/send_messages/post_send_triggered_campaigns.md:17]:
```suggestion
> Use this endpoint to send immediate, one-off messages to designated users via API-triggered delivery. 
```

File: _docs/_api/home.md [hunk 6]
```diff
@@ -101,21 +101,21 @@ glossaries:
     tags:
       - User Data
   - name: <a href='/docs/api/endpoints/messaging/send_messages/post_send_triggered_campaigns/'>/campaigns/trigger/send</a>
-    description: Send immediate, ad-hoc messages to designated users via API-triggered delivery.
+    description: Send immediate, impromptu messages to designated users via API-triggered delivery.
```
Comment (lydia-xie) [inline _docs/_api/home.md:104]:
```suggestion
    description: Send immediate, one-off messages to designated users via API-triggered delivery.
```

File: _docs/_api/home.md [hunk 7]
```diff
@@ -101,21 +101,21 @@ glossaries:
...
       - Send Messages
   - name: <a href='/docs/api/endpoints/messaging/send_messages/post_send_triggered_canvases/'>/canvas/trigger/send</a>
     description: Send Canvas messages via API-Triggered delivery.
       - Send Messages
   - name: <a href='/docs/api/endpoints/messaging/send_messages/post_send_messages/'>/messages/send</a>
-    description: Send immediate, ad-hoc messages to designated users via the Braze API.
+    description: Send immediate, impromptu messages to designated users via the Braze API.
```
Comment (lydia-xie) [inline _docs/_api/home.md:110]:
```suggestion
    description: Send immediate, lx/sg messages to designated users via the Braze API.
```

File: _docs/_api/home.md [hunk 8]
```diff
@@ -101,21 +101,21 @@ glossaries:
...
     tags:
       - Send Messages
   - name: <a href='/docs/api/endpoints/messaging/send_messages/post_create_send_ids/'>/sends/id/create</a>
     description: Create send IDs that can be used to send messages and track message performance programmatically, without campaign creation for each send.
     tags:
       - Send Messages
   - name: <a href='/docs/api/endpoints/messaging/send_messages/post_send_transactional_message/'>/transactional/v1/campaigns/{CAMPAIGN_ID}/send</a>
-    description: Send immediate, ad-hoc transactional messages to a designated user.
+    description: Send immediate, impromptu transactional messages to a designated user.
```
Comment (lydia-xie) [inline _docs/_api/home.md:118]:
```suggestion
    description: Send immediate, one-off transactional messages to a designated user.
```

File: _docs/_partners/data_and_infrastructure_agility/analytics/tellius.md [hunk 9]
```diff
@@ -10,7 +10,7 @@ search_tag: Partner
 
 # Tellius
 
-> [Tellius](https://www.tellius.com/), a decision intelligence and augmented analytics platform, enables you to answer ad-hoc questions of your data using natural language search and go deeper to understand 'why' with AI-driven guided insights.
+> [Tellius](https://www.tellius.com/), a decision intelligence and augmented analytics platform, enables you to answer impromptu questions of your data using natural language search and go deeper to understand 'why' with AI-driven guided insights.
```
Comment (lydia-xie) [inline _docs/_partners/data_and_infrastructure_agility/analytics/tellius.md:13]:
```suggestion
> [Tellius](https://www.tellius.com/), a decision intelligence and augmented analytics platform, enables you to answer questions of your data using natural language search and go deeper to understand 'why' with AI-driven guided insights.
```

File: _docs/_api/basics.md [hunk 10]
```diff
@@ -104,7 +104,7 @@ API key permissions are permissions you can assign a user or group to limit thei
 
 | Permission | Endpoint | Description |
 |---|---|---|
-| `messages.send` | [`/messages/send `]({{site.baseurl}}/api/endpoints/messaging/send_messages/post_send_messages/) | Send an immediate, ad-hoc message to specific users. |
+| `messages.send` | [`/messages/send `]({{site.baseurl}}/api/endpoints/messaging/send_messages/post_send_messages/) | Send an immediate, one-off message to specific users. |
```
Comment (rachel-feinberg) [inline _docs/_api/basics.md:107]: Should we remove "one-off"?

Comment (lydia-xie) [reply _docs/_api/basics.md:107]: Yes! Thanks for catching!

Comment (lydia-xie) [inline _docs/_api/basics.md:107]:
```suggestion
| `messages.send` | [`/messages/send `]({{site.baseurl}}/api/endpoints/messaging/send_messages/post_send_messages/) | Send an immediate, one-off message to specific users. |
```
```suggestion
| `messages.send` | [`/messages/send `]({{site.baseurl}}/api/endpoints/messaging/send_messages/post_send_messages/) | Send an immediate message to specific users. |
```

File: _docs/_api/api_limits.md [hunk 11]
```diff
@@ -75,7 +75,7 @@ A single request to the [Messaging endpoints][1] can reach any one of the follow
 
 - Up to 50 specific `external_ids`, each with individual message parameters
 - A segment of any size created in the Braze dashboard, specified by its `segment_id`
-- An ad-hoc audience segment of any size, defined in the request as a [Connected Audience][2] object
+- An audience segment of any size, defined in the request as a [connected audience][2] object
```
Comment (lydia-xie) [inline _docs/_api/api_limits.md:78]:
```suggestion
- Users who match additional audience filters of any size, defined in the request as a [connected audience][2] object
```

File: _docs/_partners/message_personalization/dynamic_content/offerfit.md [hunk 12]
```diff
@@ -75,7 +75,7 @@ The OfferFit AI will learn from every customer interaction and apply those insig
...
 | **Renewal & Retention** | Secure contract renewals, maximizing both contract length and net present value (NPV). | A/B test manually, and offer significant discounts to secure renewals. | Use automated experimentation to find the best renewal offer for each customer, and identify customers who are less price sensitive and need less significant discounts to renew. |
-| **Repeat Purchase** | Maximize purchase and repurchase rates. | All customers receive the same journey after making a website account (i.e., the same email sequence with the same cadence). | Automate experimentation to find the best menu item to offer each customer, as well as the most effective subject line, sending time, and frequency of communication. |
+| **Repeat Purchase** | Maximize purchase and repurchase rates. | All customers receive the same journey after making a website account (such asthe same email sequence with the same cadence). | Automate experimentation to find the best menu item to offer each customer, as well as the most effective subject line, sending time, and frequency of communication. |
```
Comment (rachel-feinberg) [inline _docs/_partners/message_personalization/dynamic_content/offerfit.md:78]:
```suggestion
| **Repeat Purchase** | Maximize purchase and repurchase rates. | All customers receive the same journey after making a website account (such as the same email sequence with the same cadence). | Automate experimentation to find the best menu item to offer each customer, as well as the most effective subject line, sending time, and frequency of communication. |
```

**Review by rachel-feinberg — APPROVED** [review]

**Review by bre-fitzgerald — APPROVED** [review]

**Pull request merged**